unit-tests/py/rspy/devices.py

```diff
@@ -219,8 +219,9 @@ def query( monitor_changes = True, hub_reset = False ):
         if not acroname.hub:
             acroname.connect( hub_reset )  # MAY THROW!
 
-            acroname.disable_ports( sleep_on_change = 5 )
-            acroname.enable_ports( sleep_on_change = MAX_ENUMERATION_TIME )
+            if monitor_changes:
```

Comment:
How does `monitor_changes` affect the flow of disabling and enabling the devices?

I know it doesn't change the main flow but it looks not related..

Reply:
It's on by default, and run-unit-tests leaves it on.
I.e., this affects only using devices.py thru the command-line.

Reply:
I wanted it to NOT disable and enable devices when connecting. That's what this does. Leaves the state the same as before.

Reply:
I can also add another argument...?

Reply:
I am just saying it looks like thus parameter is abused agaisnt it's name
```
param monitor_changes: If True, devices will update dynamically as they are removed/added
```

now it will also do other stuff..your call

Reply:
Changed to `recycle_ports` argument (default is True)

Reply:
In most cases you are right.
If we take into consideration that all ports were turn off on last run then we start off, and acroname reset turn all on IMO.
But if last run did not turn off then there is only a very quick power cycle (if any) to the caneras and we can get weird behaviors.
We can reduce the time after disabling from 5 to 2 for sure :-)